### PR TITLE
Add <sub> and <sup>

### PR DESCRIPTION
**Pull request by petele**

### CODE REVIEW

File: features/sub-sub.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <sub> and <sup>
-description: The `<sub>` and `<sup>` elements TODO.
+description: The `<sub>` and `<sup>` elements specifies inline text which should be displayed as subscript or superscript for solely typographical reasons.
```
Comment (foolip) [inline features/sub-sub.yml:2]: @ddbeck this runs afoul of https://github.com/web-platform-dx/web-features/tree/main/docs#specifies but "sets" wouldn't work here. Do you think this description is OK and do you want to change the guideline?

**Review by foolip — APPROVED** [review]

File: features/sub-sub.yml (hunk 1 above)
Comment (petele) [reply features/sub-sub.yml]: changed to sets

**Comment by petele** [comment]
i think this is good to merge! 😄

**Comment by foolip** [comment]
Oops, I see there's a typo in the filename. Will fix.

**Pull request merged**